doc/indices.sgm

```diff
@@ -8,15 +8,29 @@
         </title>
         <para>
           <application>pgSphere</application> uses <literal>GiST</literal>
-          to create spherical indices. An index speeds up the execution
-          time of operators <link
+          and Block Range INdexing (<literal>BRIN</literal>) algorithms to create
+          spherical indices.
+          <literal>GiST</literal> indexes utilize an R-tree implementation for
+          spherical objects, while <literal>BRIN</literal> indexes are based on "summarization"
```

Comment:
Add "the" just before `"summarization"` and re-word-wrap the whole paragraph if lines are longer than 79 characters.

Reply:
@esabol I propose not to wrap paragraphs right now because some other unchanged paragraphs do not follow this rule. It will produce more diffs to make the review process more difficult. I see that postgresql doc try to follow this rule but with tag identation is 1 characher. I have some plans to discuss with my colleagues existing best practices and adopted rules in postgresql community to create docs and to reformat pgSphere docs to follow these rules. In this task we can fix long lines.

Reply:
OK, just add the "the" then.